util/topgen.py

```diff
@@ -584,12 +584,16 @@ def generate_flash(topcfg, out_path):
     gen_rtl.gen_rtl(IpBlock.from_path(str(hjson_path), []), str(rtl_path))
 
 
-def generate_top_only(top_only_list, out_path, topname):
+def generate_top_only(top_only_dict, out_path, topname, alt_hjson_path=""):
```

Comment:
I don't think this needs a default value (since there's exactly one call-site, and you pass the argument there).

Reply:
yeah good point, i will remove.